src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/resource/SmokeTestResource.kt

```diff
@@ -45,6 +45,7 @@ class SmokeTestResource(
           hmpps-auth is providing tokens to access community-api
           communty-api allows us to update Delius
         """,
+    security = [SecurityRequirement(name = "smoke-test-security-scheme")],
```

Comment:
annoyingly the swagger parser doesn't validate the security requirements here, have to go to https://editor.swagger.io/ to find out if there are any errors